pkg/ingester/ingester.go

```diff
@@ -427,6 +452,9 @@ func (i *Ingester) updateLoop(ctx context.Context) error {
 	metadataPurgeTicker := time.NewTicker(metadataPurgePeriod)
 	defer metadataPurgeTicker.Stop()
 
+	usageStatsUpdateTicker := time.NewTicker(util.DurationWithJitter(usageStatsUpdateInterval, 0.2))
```

Comment:
I'm curious, why do we need a jitter here?

Reply:
I didn't want all ingesters do it at the same time, but in practice we don't. I will remove it.